Themis/Models/Submission.swift

```diff
@@ -45,9 +44,13 @@ struct SubmissionForAssessment: Codable {
 }
 
 extension ArtemisAPI {
-    /// Gets all submissions of that exercise.
-    static func getAllSubmissions(exerciseId: Int) async throws -> [Submission] {
```

Comment:
Don't delete

Reply:
I removed it because I didn't want dead code, but I'll use it then for the search functionality anyways, so I'll re-add 👍